package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "firstrunwizard",
   "description": "A first run wizard that explains the usage of Nextcloud to new users",
-  "version": "4.0.0",
+  "version": "5.0.0-dev.0",
```

Comment:
@nickvergessen @Pytal is the version number fine?

Reply:
Yup